package.json

```diff
@@ -40,6 +40,7 @@
     "url": "https://github.com/openmrs/openmrs-esm-login/issues"
   },
   "dependencies": {
+    "carbon-components-react": "^7.27.0",
```

Comment:
Great!

So now we only need to add `carbon-components` etc. to `peerDependencies` (shared dependencies go there, while bundled dependencies go into `dependencies`).